pkgs/tools/video/tsduck/default.nix

```
@@ -66,16 +66,22 @@ stdenv.mkDerivation rec {
     "NOPCSC=1"
     "NORIST=1"
     "NOVATEK=1"
-  ] ++ installFlags;
+  ] ++ installFlags ++ extraFlags;
 
   checkTarget = "test";
-  doCheck = true;
+  doCheck = if stdenv.isDarwin then false else true;
+
+  extraFlags = if stdenv.isDarwin then [
+    "NOTEST=1"
+    "CXXFLAGS_EXTRA=-Wno-unsafe-buffer-usage"
+  ] else [];
```

Setting `doCheck` can be shortened to:

```suggestion
  doCheck = !stdenv.isDarwin;
```

I'd avoid introducing `extraFlags` and propose to directly add the flags to `makeFlags` as necessary (see above).